src/main.cc

```diff
@@ -1443,12 +1434,55 @@ ConfigureCurrentKid(const CommandLine &cmdLine)
     }
 }
 
-static void StartUsingConfig()
+static void
```

Comment:
It would be nice to describe this function. Please adjust the text below as needed.

```suggestion
/// Start directing debugs() messages to the configured cache.log.
/// Until this function is called, all allowed messages go to stderr.
static void
```